apps/site/src/app/admin/participants/components/ParticipantsTable.tsx

```diff
@@ -9,9 +9,14 @@ import TextFilter from "@cloudscape-design/components/text-filter";
 import ApplicantStatus from "@/app/admin/applicants/components/ApplicantStatus";
 import { Participant } from "@/lib/admin/useParticipants";
 
+import CheckinDayIcon from "./CheckinDayIcon";
 import ParticipantAction from "./ParticipantAction";
 import RoleBadge from "./RoleBadge";
 
+const FRIDAY = new Date("2024-01-26T12:00:00");
+const SATURDAY = new Date("2024-01-27T12:00:00");
+const SUNDAY = new Date("2024-01-28T12:00:00");
```

Comment:
Do we need to use UTC here to prevent potential timezone issues like in the schedule countdown?

Reply:
The timezones are encoded within the `CheckinDayIcon`, and since the time of noon is used, any timezone shift should still be within the same day.